.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Is it important for this file to be checked in?

Reply:
Nope, this is just a file to my source control operates, but need to delete before the merge

Reply:
for future reference, if you don't want this checked in and don't want to deal w/ it in the commit status, you should add it to the `.gitignore` file!